webhook/Makefile

```diff
@@ -0,0 +1,18 @@
+NAME = javaagent-webhook
+IMAGE_PREFIX = hypertrace
```

Comment:
nit: `DOCKER_NAMESPACE`

also make all the env vars configurable - `?= hypertrace`

Reply:
fixed